src/Elcodi/Admin/MetricBundle/AdminMetricBundle.php

```diff
@@ -17,16 +17,15 @@
 
 namespace Elcodi\Admin\MetricBundle;
 
-use Symfony\Component\Console\Application;
-use Symfony\Component\CssSelector\XPath\Extension\ExtensionInterface;
-use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 
 use Elcodi\Admin\MetricBundle\DependencyInjection\AdminMetricExtension;
+use Elcodi\Bundle\CoreBundle\Abstracts\AbstractElcodiBundle;
 
 /**
  * Class AdminMetricBundle
  */
-class AdminMetricBundle extends Bundle
+class AdminMetricBundle extends AbstractElcodiBundle
```

Comment:
done, sorry. I started this pull request with totaly other idea and then I saw the AbstractElcodiBundle class and refactored ^^